### PR TITLE
HUD Aviators Fix - Flash Protection & Sprite 

### PR DESCRIPTION
**Pull request by WickedCybs**
The HUD aviators now provide flash protection in flash protection mode. The flash protection mode sprite has also been updated, its left and right facing sprites were identical to HUD mode, appearing light when it should have been slightly dark.

guMarch2021Emeraldtreeskink

gsMarch2021Tayra

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Comment by WickedCybs** [comment]
!bugfix

**Comment by WickedCybs** [comment]
Will have fixed my own issue with this.

https://github.com/Aurorastation/Aurora.3/issues/10034

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -593,8 +594,8 @@ BLIND     // can't see anything
 			src.hud = hud_holder
 			to_chat(user, "You switch \the [src] to HUD mode.")
 		else
-			flash_protection = initial(flash_protection)
-			active = 0
+			flash_protection = FLASH_PROTECTION_MODERATE
+			active = 1
```
Comment (NonQueueingMatt) [inline code/modules/clothing/glasses/glasses.dm:598]:
```suggestion
			active = TRUE
```

**Review by Geevies — APPROVED** [review]

**Pull request merged**